central/baseimage/watcher/pattern_matcher.go

```diff
@@ -0,0 +1,34 @@
+// Package watcher provides base image repository watching functionality for tag discovery.
+package watcher
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	"github.com/stackrox/rox/pkg/registries/types"
+)
+
+// listAndFilterTags lists all tags for the given image and filters them using
+// the using [path.Match] glob pattern. It returns the list of matching tags or
```

Comment:
nit: Remove "using the"

Reply:
Fixed. Let me know if it looks OK by resolving this conversation.